common/testing/historyrequire/history_require.go

```diff
@@ -76,7 +76,9 @@ func (h HistoryRequire) EqualHistoryEvents(expectedHistory string, actualHistory
 
 	expectedHistoryEvents, expectedEventsAttributes := h.parseHistory(expectedHistory)
 
-	require.Equalf(h.t, len(expectedHistoryEvents), len(actualHistoryEvents), "Length of expected(%d) and actual(%d) histories is not equal", len(expectedHistoryEvents), len(actualHistoryEvents))
+	require.Equalf(h.t, len(expectedHistoryEvents), len(actualHistoryEvents),
+		"Length of expected(%d) and actual(%d) histories is not equal: \n%v",
```

Comment:
```suggestion
		"Length of expected(%d) and actual(%d) histories is not equal:\nActual history:\n%v",
```
and everywhere bellow.